lib/yang.c

```diff
@@ -393,6 +393,23 @@ const char *yang_dnode_get_schema_name(const struct lyd_node *dnode,
 	return dnode->schema->name;
 }
 
+struct lyd_node *yang_dnode_new_path(struct lyd_node *dnode,
+				     const char *xpath_fmt, ...)
```

Comment:
Add `PRINTF_ATTRIBUTE(2, 3)` so the compiler checks format strings please

Reply:
Done in the header.